File: go/muse/algorithms/sorting/sorting.go
package sorting

import (
	"muse/algorithms/sorting/bubble_sort"
	"muse/algorithms/sorting/insertion_sort"
	"muse/algorithms/sorting/merge_sort"
	"muse/algorithms/sorting/quick_sort"
	"muse/algorithms/sorting/selection_sort"
)

func BubbleSort(arr []int) {
	bubble_sort.Sort(arr)
}

func InsertionSort(arr []int) {
	insertion_sort.Sort(arr)
}

func MergeSort(arr []int) {
	merge_sort.Sort(arr)
}

func QuickSort(arr []int) {
	quick_sort.Sort(arr)
}

func SelectionSort(arr []int) {
	selection_sort.Sort(arr)
}

File: python/muse/algorithms/sorting/sorting.py
from muse.algorithms.sorting import (
    bubble_sort,
    insertion_sort,
    merge_sort,
    quick_sort,
    selection_sort,
)


def bubblesort(arr: list[int]) -> None:
    bubble_sort.sort(arr)


def insertionsort(arr: list[int]) -> None:
    insertion_sort.sort(arr)


def mergesort(arr: list[int]) -> None:
    merge_sort.sort(arr)


def quicksort(arr: list[int]) -> None:
    quick_sort.sort(arr)


def selectionsort(arr: list[int]) -> None:
    selection_sort.sort(arr)
